### ISSUE TITLE
[A11y] Desktop Work Show Page-CPD Parent: Universal Viewer

### ISSUE THREAD

**Issue by azle** [OregonDigital/OD2#1971]
### Operating System

MacOS

### Browser

Chrome

### Affected page

Work Show

### Admin user?

No

### Issue Description

Work Show Page-CPD Parent: https://staging.oregondigital.org/concern/generics/df727084j?locale=en

The universal viewer has a link in the upper left corner of the viewer and it doesn't have a label. When I tab to it and click enter it takes me to the homepage.

See screensnap
<img width="1188" alt="Screen Shot 2021-08-24 at 10 57 05 AM" src="https://user-images.githubusercontent.com/8353860/130667060-547de9b2-b281-4b16-a999-7fc4874ed5ad.png">

### Suggested Fix

Remove the link or at least give it a label.

### Can anybody QA or does this need an expert?

A11y familiarity recommended

**Comment by azle** [comment]
Updated 12/30/2024 FIXED

**Comment by azle** [comment]
Updated 1/6/2024 FIXED
